pipeline-scripts/buildlib.groovy

```diff
@@ -724,6 +724,27 @@ def build_ami(major, minor, version, release, yum_base_url, ansible_branch, mail
     }
 }
 
+def sweep(build_version, sweep_builds) {
+    def sweep_job = build(
+        job: 'build%2Fsweep',
+        propagate: false,
+        parameters: [
+            param('String', 'BUILD_VERSION', build_version),
+            param('Boolean', 'SWEEP_BUILDS', sweep_builds),
+        ]
```

Comment:
@thiagoalessio You didn't test your PR, right?
It seems to me this would cause a syntax error just like the one addressed by https://github.com/openshift/aos-cd-jobs/pull/2064.

Go fixing it!

Reply:
fixed.